### DISCUSSION TITLE
full backup of the new installed and configured instance possible?

### DISCUSSION THREAD

**Discussion by h3rm** [Guilouz/Klipper-Flsun-Speeder-Pad#153] [Q&A]
After a few days configuring and calibrating everything is configured now and the v400 works great with the original klipper! big thx + donation is out! :)

My question: Is it possible to clone the hole instance (all files + configurations on the internal SD) to a backup image? Maybe

Why do i ask?: If i maybe brick something in the future it would be nice if the hole flashing procedure doesn't have be done again and a full backup of "everything" would be helpful in this case.

I already saw that there are some gits to automatically backup the config-files. but i couldn't find out if there is a way to to a "full backup" which can be flashed back if something gets broken

**Comment by Barry4u** [comment]
No answer here but I am very interested in a good answer.

I just COMPLETED the full stock install myself and am very concerned about having to redo everything. 

I am pleased with the performance of the V600 now.

Just checked AND  there is a Klipper update and 9 packages can be upgraded.

I too would like to be able to take a snapshot & restore it in case something goes wrong. 

Thank you.
